examples/kurth/input_kurth_10nC.in

```diff
@@ -0,0 +1,37 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000
+beam.units = static
+beam.energy = 2.0e3
+beam.charge = 1.0e-8
+beam.particle = proton
+beam.distribution = kurth6d
+beam.sigmaX = 1.2154443728379865788e-3
```

Comment:
double check vs. docs: Are LHS of `amrex::ParmParse` (keys) case-sensitive?

Reply:
The logic for those params should not be case insensitive, but we use this already in:
- `fodo_rf`
- `multipole`
- `chicane`
- `kurth`
- ...
:thinking:

Reply:
Oh, I think it's a documentation mistake in ImpactX

Reply:
-> #258